internal/config/const.go

```diff
@@ -29,4 +29,6 @@ const (
 	privateAPIKey       = "private_api_key"
 	opsManagerURL       = "ops_manager_url"
 	baseURL             = "base_url"
+	LabelKey            = "Infrastructure Tool"
```

Comment:
[nit] this is not part of the cli configuration so let's keep it in the cli package